hnn_core/extracellular.py

```diff
@@ -427,6 +429,63 @@ def plot(self, *, trial_no=None, contact_no=None, tmin=None, tmax=None,
                 show=show)
         return fig
 
+    def csd_plot(self,
+                 trial_no: int = 0,
+                 method: Literal["delta", "step", "spline",
+                                 "standard"] = "standard",
+                 f_type="gaussian",
+                 filtered: bool = True,
```

Comment:
we don't use type annotations in `hnn-core` yet. I think this would be worth discussing at a future point but for now, I recommend to prefer consistency within codebase

Reply:
I see, for compatibility and consistency I'll remove it later